channel/state/outcome/guarantee.go

```diff
@@ -0,0 +1,60 @@
+package outcome
+
+import (
+	"encoding/json"
+	"log"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/statechannels/go-nitro/types"
+)
+
+// A Guarantee is an Allocation with AllocationType == GuaranteeAllocationType and Metadata = encode(GuaranteeMetaData)
+type GuaranteeMetadata struct {
+	Left  types.Address // The peer who plays the role of Alice (peer 0)
+	Right types.Address // The peer who plays the role of Bob (peer n+1, where n=len(peers))
```

Comment:
Is there a case for these to be `types.Destination`s instead?

Reply:
I think yes.

Reply:
eee4945